src/MISC/fix_deposit.cpp

```diff
@@ -193,6 +193,14 @@ FixDeposit::FixDeposit(LAMMPS *lmp, int narg, char **arg) :
   next_reneighbor = update->ntimestep + 1;
   nfirst = next_reneighbor;
   ninserted = 0;
+
+  // throw away the first few numbers to avoid the unexpected correlations 
```

Comment:
Please put this code immediately after the generator is constructed.
And make the comments be:
// random number generator, same for all procs
// warm-up the generator 30x to avoid correlations in first-particle positions if
// runs are repeated with consecutive seeds